ReactAndroid/src/androidTest/js/ScrollViewTestModule.js

```diff
@@ -37,54 +36,52 @@ class Item extends React.Component {
   }
 }
 
-var getInitialState = function() {
-  var data = [];
-  for (var i = 0; i < NUM_ITEMS; i++) {
-    data[i] = {text: 'Item ' + i + '!'};
+class ScrollViewTestApp extends React.Component {
+  constructor() {
+    super();
```

Comment:
React components are instantiated with arguments (the first of which is just `props`). To avoid potential problems, we should just forward them all to `React.Component` constructor.

```JavaScript
class ScrollViewTestApp extends React.Component {
  constructor(...args) {
    super(...args);
    // ...
  }
  // ...
}
```